content/blog/2017/12/2017-12-11-auto-convert-freestyle-jenkins-jobs-to-coded-devops-pipeline.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Auto-Convert Freestyle Jenkins Jobs to Coded DevOps Pipeline" 
+tags:
+- plugin
+- pipeline
+author: alyssat
+---
+
+[NOTE]
+====
+This is a guest post by Sanil Pillai, Director of Labs & Strategic Insights, Infostretch
+====
+
+link:https://www.infostretch.com[Infostretch] has created a link:https://plugins.jenkins.io/convert-to-pipeline[plugin] for DevOps teams migrating from Jenkins 1.0 to Jenkins 2.0 that automatically converts freestyle jobs to coded pipeline. This new plugin streamlines the process and accelerates pipeline on-boarding for any new set of applications. Previously, when migrating to Jenkins 2.0, converting freestyle jobs required DevOps engineers to drill down on each one of those hundreds or thousands of jobs to understand tools, configurations, URLs, parameters and more before rewriting them in pipeline syntax. This process is very manual, error-prone, lengthy and not cost-effective. Beyond saving time, the new plugin also assures adherence to proper coding standards and separates complex business logic and standards declaration from execution flow. 
```

Comment:
No line breaks. I'd says that "Jenkins 1.0" and "Jenkins 2.0" should not be used in this context